### ISSUE TITLE
Map icon dont dissapear

### ISSUE THREAD

**Issue by FlatErik89** [Jaliborc/PetTracker#347]
**Running Software (issues missing this information will be deleted):**
 - Addon version: PetTracker-10.0.1 for WoW-10.0.2.zip
 - Server patch: 10.0.5.48069

 **Have you read the changelog? (please don't waste our time)**
 Yes

**Describe the bug**
Even if i had capture "Rusty Snail" and "Rabid Nut Varmint 5000", both of them still apear at the map when i select left species.

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**
I expect the to disapear from the map when i own them 

**Screenshots**

**Error Logs**
Type `/console scriptErrors 1` in the chat and reload the game. If an error window appears, write here the message with `ID: 1` (labelled 1/X). This is generally NOT the message that the window shows by default.

**Additional context**
![caracol](https://user-images.githubusercontent.com/126237311/221132637-7b984350-15eb-47c0-9db2-59f4d686c709.png)
![Alimaña](https://user-images.githubusercontent.com/126237311/221132643-97d9c8f0-db82-4324-a698-472631adcbca.png)

**Comment by GurliGebis** [comment]
Have you tried filtering for just missing pets?

**Comment by FlatErik89** [comment]
Yup, it is on, u can see the cross at the right. Only that idk why it doesn`t appear

**Comment by Rezy942** [comment]
Perhaps not related to this problem - but the settings part of this addon is a complete mess imo.
Settings scattered between options and map - some of which are unintuitive and confusing when it comes to multiple characters.
